<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::CookiePropertiesDialog</class>
 <widget class="QDialog" name="Otter::CookiePropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
       <property name="buddy">
        <cstring>nameLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="valueLabel">
       <property name="text">
        <string>Value:</string>
       </property>
       <property name="buddy">
        <cstring>valueLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="valueLineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="expiresLabel">
       <property name="text">
        <string>Expires:</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
       <property name="buddy">
        <cstring>expiresDateTimeEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="isSessionOnlyCheckBox">
       <property name="text">
        <string>this session only</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDateTimeEdit" name="expiresDateTimeEdit">
       <property name="displayFormat">
        <string comment="Date and time format">MM.dd.yyyy HH:mm</string>
       </property>
       <property name="timeSpec">
        <enum>Qt::UTC</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="domainLabel">
       <property name="text">
        <string>Domain:</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
       <property name="buddy">
        <cstring>domainLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="domainLineEdit">
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="pathLabel">
       <property name="text">
        <string>Path:</string>
       </property>
       <property name="buddy">
        <cstring>pathLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="pathLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="isSecureCheckBox">
     <property name="text">
      <string>Send only for secure connections</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="isHttpOnlyCheckBox">
     <property name="text">
      <string>Allow accessing using JavaScript</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>valueLineEdit</tabstop>
  <tabstop>domainLineEdit</tabstop>
  <tabstop>pathLineEdit</tabstop>
  <tabstop>isSessionOnlyCheckBox</tabstop>
  <tabstop>expiresDateTimeEdit</tabstop>
  <tabstop>isSecureCheckBox</tabstop>
  <tabstop>isHttpOnlyCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Otter::CookiePropertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Otter::CookiePropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
